tower/src/hedge/rotating_histogram.rs

```diff
@@ -19,10 +19,10 @@ pub struct RotatingHistogram {
 impl RotatingHistogram {
     pub fn new(period: Duration) -> RotatingHistogram {
         RotatingHistogram {
-            read: Histogram::<u64>::new_with_bounds(1, 10_000, 3)
-                .expect("Invalid histogram params"),
-            write: Histogram::<u64>::new_with_bounds(1, 10_000, 3)
-                .expect("Invalid histogram params"),
+            // Use an auto-resizing histogram to avoid choosing
+            // a maximum latency bound for all users.
```

Comment:
maybe worth a note that this means it will resize automatically? not a blocker.

Reply:
sorry, just to clarify you mean a note that `::new` creates an auto-resizing histogram, or?

Reply:
never mind, i can't read